src/m365/spo/commands/list/list-sensitivitylabel-ensure.ts

```diff
@@ -0,0 +1,149 @@
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import request, { CliRequestOptions } from '../../../../request';
+import { formatting } from '../../../../utils/formatting';
+import { urlUtil } from '../../../../utils/urlUtil';
+import { validation } from '../../../../utils/validation';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+export interface Options extends GlobalOptions {
+  webUrl: string;
+  name: string;
+  listId?: string;
+  listTitle?: string;
+  listUrl?: string;
+}
+
+class SpoListSensitivityLabelEnsureCommand extends SpoCommand {
+  public get name(): string {
+    return commands.LIST_SENSITIVITYLABEL_ENSURE;
+  }
+
+  public get description(): string {
+    return 'Sets a default sensitivity label to the specified document library.';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        listId: (!(!args.options.listId)).toString(),
```

Comment:
Let's use `typeof args.options.listId != undefined`

(Or something like that 😀 )